orchestration/dagster_orchestration/hca_manage/check.py

```diff
@@ -1,9 +1,19 @@
 import argparse
-from typing import Optional
+import csv
+from dataclasses import dataclass
+import logging
+import os
+from typing import Callable, Optional, TextIO
+
+from cached_property import cached_property
```

Comment:
i screwed up in pioneering this pattern, but in the event that we want to cache properties, `cached_property` is now part of python core, under `functools`